css/index.css

```diff
@@ -0,0 +1,387 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&family=Urbanist:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: Poppins;
+}
+
+.wrapper {
+  width: 375px;
+  background-color: #e5e5e5;
+  display: grid;
+}
```

Comment:
[OPTIONAL] 
- [ ] Nice work, your design looks almost identic to Figma's design :1st_place_medal: But please add some space at the top of your "works" section (see images)

### Your website
![Screenshot from 2023-03-27 14-39-16](https://user-images.githubusercontent.com/91142432/227979816-4a869bdc-99de-445b-aacf-f5503992c277.png)

### What it suppose to look like 
![Screenshot from 2023-03-27 14-41-52](https://user-images.githubusercontent.com/91142432/227980031-9965200c-967b-48ad-85fb-c60cea24b0ca.png)

This is a requirement of the project but the previous reviewers didn't notice it. so I requested this as an optional but I highly recommend you to make the change